org.hl7.fhir.r5/src/main/java/org/hl7/fhir/r5/terminologies/expansion/PropertyFilter.java

```diff
@@ -28,7 +29,7 @@ public PropertyFilter(List<String> allErrors, ConceptSetFilterComponent fc, Prop
   public boolean includeConcept(CodeSystem cs, ConceptDefinitionComponent def) {
     ConceptPropertyComponent pc = getPropertyForConcept(def);
     if (pc != null) {
-      String v = pc.getValue().isPrimitive() ? pc.getValue().primitiveValue() : null;
+      String v = pc.getValue().isPrimitive() ? pc.getValue().primitiveValue() : pc.getValue() instanceof Coding ? ((Coding) pc.getValue()).getSystem()+"#"+((Coding) pc.getValue()).getCode() : null;
```

Comment:
The nesting here is getting a little out of hand for a single line.

Maybe this:

```java
private static String getStringValueFromProperty(ConceptPropertyComponent propertyForConcept) {
    if (propertyForConcept.getValue().isPrimitive()) {
      return propertyForConcept.getValue().primitiveValue();
    }
    if (propertyForConcept.getValue() instanceof Coding coding) {
      return coding.getSystem() + "#" + coding.getCode();
    }
    return null;
  }
```